### DISCUSSION TITLE
Is the application using the header file be applied to the GPL license?

### DISCUSSION THREAD

**Discussion by takemuun** [umlaeute/v4l2loopback#559] [Q&A] [answered]
Hi,

If I develop an application using the following header file, will the created application be applied to the GPL license?

https://github.com/umlaeute/v4l2loopback/blob/main/v4l2loopback.h

Thank you.

**Comment by umlaeute** [comment, marked as answer]
the license of the `v4l2loopback.h` header is `GPL-2.0+ WITH Linux-syscall-note`: https://github.com/umlaeute/v4l2loopback/blob/5bb9bed4e2a34761dc5dc188faec7315ae3930fe/v4l2loopback.h#L1

The `Linux-syscall-note` exception can be found at https://spdx.org/licenses/Linux-syscall-note.html
the relevant part is:
> NOTE! This copyright does *not* cover user programs that use kernel services by normal system calls - this is merely considered normal use of the kernel, and does *not* fall under the heading of "derived work". 

as such, any program *using* the header in order to issue syscalls, is not covered by the GPL-2.0+

see also https://opensource.stackexchange.com/q/6647
